cdap-ui/app/directives/widget-container/widget-complex-schema-editor/widget-complex-schema-editor.js

```diff
@@ -197,6 +197,14 @@ function ComplexSchemaEditorController($scope, EventPipe, $timeout, myAlertOnVal
           return;
         }
 
+        // TODO: for splitters, the backend returns port names similar to [schemaName].string or [schemaName].int.
```

Comment:
nit: Lets have TODOs with JIRA so that we can track and fix it. JIRA would be a good place to add context to things.

Reply:
Done.